docs/src/main/sphinx/release/release-463.md

```diff
@@ -0,0 +1,35 @@
+# Release 463 (23 Oct 2024)
+
+## General
+
+* Enable HTTP/2 for internal communication by default. ({issue}`21793`)
```

Comment:
@wendigo do we also want to add that HTTP/2 support for clients is there now? Also .. any config we need to add to the release notes as fallback or do we want to link somewhere?

Reply:
Maybe just say "the previous behavior can be restored by setting xxx to yyy".

Reply:
Ok .. I will also send a docs PR but ideally we get #23832 and #23833 merged as well so I can cleanly add a new page for internal communication properties.